### DISCUSSION TITLE
How can I use your source files for chatgpt-advanced.

### DISCUSSION THREAD

**Discussion by Dubudubap** [qunash/chatgpt-advanced#105] [General]
Hello.
I got a big inspiration from your project.
I am a university student who is doing a project about chat-gpt.
And your real-time chat function idea is amazing.
I'd like to vary your idea to get some simple infos from a website.
But I am not sure how can I make it based on your idea.
Can I get some informations how did you do this?

**Comment by qunash** [comment]
Hi, the code is MIT licensed, see https://github.com/qunash/chatgpt-advanced/blob/main/LICENSE
